GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -2907,6 +2909,8 @@ subroutine get_points_weights_tau(grid_size, e_range, ac_we, ierr)
     !> Internal variables
     integer                                                :: ien, kloc, bup
     type(er_aw_aux)                                        :: aw
+    real(kind=dp)                                          :: e_ratio
+
 
     !> Begin work
     ierr = 0
```

Comment:
One could rework this logic a bit to be:

```fortran
if (.not. iany(tau_npoints_supported == grid_size)) then
       ierr = 1
       _REGISTER_EXC("The grid size you chose is not available.")
       return
endif

Then the rest of the if statement can be scrapped
```

Reply:
Nice suggestion, I will also implement it. I had to add a test for the AC anyway, so I will include it in the next PR